x-pack/plugins/security_solution/public/plugin.tsx

```diff
@@ -155,27 +155,27 @@ export class Plugin implements IPlugin<PluginSetup, PluginStart, SetupPlugins, S
       return services;
     })();
 
-    core.application.register({
-      id: `${APP_ID}:${SecurityPageName.detections}`,
-      title: DETECTION_ENGINE,
-      order: 9001,
-      euiIconType: APP_ICON_SOLUTION,
-      category: DEFAULT_APP_CATEGORIES.security,
-      appRoute: APP_DETECTIONS_PATH,
-      updater$: this.detectionsUpdater$,
-      mount: async (params: AppMountParameters) => {
-        const [coreStart, startPlugins] = await core.getStartServices();
-        const { detections: subPlugin } = await this.subPlugins();
-        const { renderAppOld } = await this.lazyApplicationDependencies();
-
-        return renderAppOld({
-          ...params,
-          services: await startServices,
-          store: await this.store(coreStart, startPlugins),
-          SubPluginRoutes: subPlugin.start(this.storage).SubPluginRoutes,
-        });
-      },
-    });
+    // core.application.register({
+    //   id: `${APP_ID}:${SecurityPageName.detections}`,
+    //   title: DETECTION_ENGINE,
+    //   order: 9001,
+    //   euiIconType: APP_ICON_SOLUTION,
+    //   category: DEFAULT_APP_CATEGORIES.security,
+    //   appRoute: APP_DETECTIONS_PATH,
+    //   updater$: this.detectionsUpdater$,
```

Comment:
we'll have to migrate the `updater$` of each plugin to a generic `updater$: this.appUpdater$` in the main application